src/App.js

```diff
@@ -1,13 +1,62 @@
 import React from "react";
 import "./App.css";
 
+
 class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+        score: 0,
+        increment: 1
+    }
+  }
+
+  incrementMethod = () => {
+    const {score, increment} = this.state; 
+    this.setState({
+        score: score + increment
+    })
+  }
```

Comment:
```suggestion
  incrementMethod = () => {
      this.setState((prevState) => {
      score: prevState.score + prevState.increment
      }
  }
```
The way you have incrementMethod written already **is fine in this case**, but I just wanted to show you that this is also an option! It's a little more advanced and not as intuitive to beginners, so we haven't touched on it in class yet. setState can accept a callback function that has the previous state as a parameter. Depending on how your logic is written, in some cases, using setState this way is actually necessary.

Here's the documentation on using setState this way: https://reactjs.org/docs/faq-state.html